apps/web/src/App.tsx

```diff
@@ -48,10 +48,11 @@ if (SENTRY_DSN) {
   });
 }
 
-const defaultQueryFn = async ({ queryKey }: { queryKey: string }) => {
-  const response = await api.get(`${queryKey[0]}`);
+// eslint-disable-next-line no-console
+console.log('trigger web ci test');
```

Comment:
I think that left here by mistake :)

Reply:
hahah yes did it in the first place in order to trigger the ci test, I will remove it.
I probably will create a new PR with rebased commit, this PR is too messy in my opinion.